src/components/KCard/KCard.vue

```diff
@@ -122,13 +132,17 @@ export default defineComponent({
     },
   },
 
-  setup(props) {
+  setup(props, { slots }) {
     const titleId = computed((): string => props.testMode ? uuidv1() : 'test-title-id-1234')
     const contentId = computed((): string => props.testMode ? uuidv1() : 'test-content-id-1234')
+    const useStatusHatLayout = computed((): boolean => {
+      return !!(props.status || slots.statusHat)
```

Comment:
```suggestion
      return !!(props.status || !!slots.statusHat)
```

I "think" this needs to actually look like this